### PR TITLE
Set variables right

### PR DESCRIPTION
**Pull request by MemoICT**
### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Docs/Resources/current/4-how-to/130-creating-entities-dal.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ $taxRepository->upsert(
 /** @var EntityRepositoryInterface $taxRepository */
 $taxRepository = $this->container->get('tax.repository');
 
-$repository->upsert(
+$taXepository->upsert(
```
Comment (mitelg) [inline src/Docs/Resources/current/4-how-to/130-creating-entities-dal.md]:
```suggestion
$taxRepository->upsert(
```

**Comment by mitelg** [comment]
hey @MemoICT 
thanks for willing to take part in our documentation :blush: although it is just typo fixing, could you please fill out the PR template at least a little? :wink:

**Comment by MemoICT** [comment]
Ok did this.

**Comment by mitelg** [comment]
thanks for your contribution @MemoICT :+1: :tada: :blue_heart:

**Pull request closed without merging**